tv_shows_app/src/components/features/reviews/ReviewItem/ReviewItem.tsx

```diff
@@ -69,15 +78,10 @@ export const ReviewItem = ({ review, onRemove }: IReviewItemProps) => {
         </Flex>
       </Flex>
       <Text fontSize="xs">{review.comment}</Text>
-      <IconButton
-        backgroundColor="orange.100"
-        _hover={{ backgroundColor: 'red.300' }}
-        marginLeft="auto"
-        aria-label="Delete review"
-        size="sm"
-        icon={<DeleteIcon />}
-        onClick={onClickHandler}
-      />
+      <Flex marginLeft="auto" direction="row" gap="3">
+        <EditReviewSection review={review} onEdit={onEdit} />
+        <DeleteButton review={review} onRemove={onRemove} />
```

Comment:
Rekli smo ovdje mozemo dodati conditional logiku za prikaz ovih button-a:
Ja osobno bih izvan return-a (negdje u komponenti) kreirao varijablu:

(nekako moras doci do logirano user-a) -> hint: spremljen je u cache-u negdje (gdje si to definirao s loginom)
`const isMyReview = loggedInUser.email === review.user.email;`

i onda dolje imao:
```typescript
{isMyReview && (
  <>
    <EditReviewSection review={review} onEdit={onEdit} />
    <DeleteButton review={review} onRemove={onRemove} />
  </>
)}
```